modules/components/RouteHandler.js

```diff
@@ -42,7 +42,10 @@ class RouteHandler extends React.Component {
   }
 
   render() {
-    return this.createChildRouteHandler();
+    // IMPORTANT: This span "soaks" up owner context, keeping
+    // React 0.13.0 from throwing a warning.
+    // TODO: Why should this even be necessary?
+    return <span>{ this.createChildRouteHandler() }</span>
```

Comment:
We should _really_ talk this part through. Without this span, RouteHandlers without a parent will cause React to throw the `owner vs parent` context warning.